com.anydesk.Anydesk.json

```diff
@@ -28,6 +28,25 @@
                 }
             ]
         },
+        {
+            "name": "zlib",
+            "build-commands": [
+                "./configure --prefix=/app",
+                "make",
+                "make install",
+                "cd ./contrib/minizip && autoreconf --install",
+                "cd ./contrib/minizip && ./configure --prefix=/app",
+                "cd ./contrib/minizip && make",
+                "cd ./contrib/minizip && make install"
+            ],
```

Comment:
zlib is included  in the freedesktop runtime – indeed it's exactly the same version, 1.2.11. I think you can just build the `contrib/minizip` subdir.

See https://github.com/flathub/org.qgis.qgis/blob/d94ead7cbba1123e9a467a04391e866e27924e3a/org.qgis.qgis.json#L278-L295 https://github.com/flathub/org.wireshark.Wireshark/blob/b52d415985f08a7a0673ee3efb4e5f59a1193aeb/org.wireshark.Wireshark.yml#L22-L36 for examples.

Reply:
@wjt thank you for the examples. Updated the code as requested. Latest build doesn't seem to have issues.